ui/inspectors/virtual-machine.reel/virtual-machine.html

```diff
@@ -29,7 +29,8 @@
             "bindings": {
                 "context": {"<-": "@owner.context"},
                 "canDelete": {"<-": "!@owner.object._isNew"},
-                "classList.has('has-loading-spinner')": {"<-": "@owner.isLoading"}
+                "classList.has('has-loading-spinner')": {"<-": "@owner.isLoading"},
```

Comment:
@pchaussalet If I remember well there is already a Class Name `has-loading-spinner` or something else within the inspector when the property `isLoading` is `true`?